cpp/arduino/avr/CPPLINT.cfg

```diff
@@ -0,0 +1 @@
+exclude_files=.*\.h
```

Comment:
If this config file format allows comments, can you add some explicit text to describe what is being ignored here & why?

Reply:
Yes, that would be helpful.